a3p-integration/proposals/n:upgrade-next/initial.test.js

```diff
@@ -10,7 +10,7 @@ const vats = {
   orchestration: { incarnation: 1 },
   transfer: { incarnation: 2 },
   walletFactory: { incarnation: 6 },
-  zoe: { incarnation: 4 },
+  zoe: { incarnation: 3 },
```

Comment:
how did this number go down?

Reply:
I think because `upgrade.go` somehow kept some u18 proposals around when it was rebased on top of u18.